### PR TITLE
Update submit_feedback view

### PR DESCRIPTION
**Pull request by carsondrobe**
This pull request updates the submit_feedback view to save the feedback to the database and adds tests for the view.

### CODE REVIEW

**Review by carsondrobe — COMMENTED** [review]
Created a new issue for linking feedback to accounts. Until Users/Login are setup, all feedback submitted is defaulting to sent by admin

File: Site/bc_weather_station_dashboard/website/migrations/0002_alert_dashboard_feedback_responsefromadmin_and_more.py
Comment (robertyac) [inline Site/bc_weather_station_dashboard/website/migrations/0002_alert_dashboard_feedback_responsefromadmin_and_more.py:1]: Migration? Should this be included? I'm pretty sure these are like the .pyc files?

File: Site/bc_weather_station_dashboard/website/migrations/0003_alter_feedback_table.py
Comment (robertyac) [inline Site/bc_weather_station_dashboard/website/migrations/0003_alter_feedback_table.py:1]: LGTM

File: Site/bc_weather_station_dashboard/website/models.py
Comment (robertyac) [inline Site/bc_weather_station_dashboard/website/models.py:1]: What are you changing here? just the ' and " ?

File: Site/bc_weather_station_dashboard/website/templates/home.html
Comment (robertyac) [inline Site/bc_weather_station_dashboard/website/templates/home.html:1]: LGTM

File: Site/bc_weather_station_dashboard/website/views.py
Comment (robertyac) [inline Site/bc_weather_station_dashboard/website/views.py:1]: LGTM

**Comment by robertyac** [comment]
What I know you changed LGTM. I'm just having a hard time seeing what files you explicitly changed, I think in the meeting you changed the view file? Is vscode just auto updating all those other files?

**Comment by carsondrobe** [comment]
Yes, views, tests, and home.html. The other database stuff is from migrating to the updated DB names and stuff. It all still runs as expected on my machine but before approving you should try running it on your machine. Check both the regular dashboard and Admin Database side, please.

**Comment by robertyac** [comment]
> git checkout -q --track origin/76-send-feedback-to-database
error: The following untracked working tree files would be overwritten by checkout:
	Site/bc_weather_station_dashboard/website/migrations/0002_alert_dashboard_feedback_responsefromadmin_and_more.py
Please move or remove them before you switch branches.
Aborting


Migrations shouldn't be uploaded I dont think? It's giving me this when I try to switch branches.

Maybe try starting a new branch and only change the files you changed and submit a pull request after you stage only those files?

**Comment by carsondrobe** [comment]
should work now

**Comment by carsondrobe** [comment]
Should we add /migrations folder to the .gitignore? This is the directory containing all the .pyc files and the migration problems

**Comment by robertyac** [comment]
Yeah that would be a good idea

**Comment by carsondrobe** [comment]
Nevermind, from researching we do need it

**Review by robertyac — APPROVED** [review]
looks good. I deleted my 0002 migration to switch to your branch otherwise it wouldn't allow me. 

Now I need that 0002 migration back or I can't run the server and stuff.

**Review by cmcahoon01 — APPROVED** [review]
I am unclear what you are doing in 0002_alert_dashboard_feedback_responsefromadmin_and_more.py
but if you think that file is necessary, everything else appears to be working fine.

**Pull request merged**